R/derive_params_growth_age.R

```diff
@@ -236,7 +238,10 @@ derive_params_growth_age <- function(dataset,
       by = c("sex_join", "ageu_join"),
       relationship = "many-to-many"
     ) %>%
-    filter(prev_age <= {{ age }} & {{ age }} < next_age)
+    mutate(age_diff := abs(metadata_age - {{ age }})) %>%
+    group_by(USUBJID) %>%
+    mutate(is_lowest = age_diff == min(age_diff)) %>%
+    filter(is_lowest)
```

Comment:
we should explain this nearest matching approach in both function roxygen2 headers sections so users are completely clear on how cases are handling when fallimng in between 2 metadata rows